docs/prod-install.md

```diff
@@ -39,27 +39,33 @@ with the following commands:
 
 ```
 sudo -i  # If not already root
-cd /root
+cd $(mktemp -d)
 wget https://www.zulip.org/dist/releases/zulip-server-latest.tar.gz
-rm -rf /root/zulip && mkdir /root/zulip
-tar -xf zulip-server-latest.tar.gz --directory=/root/zulip --strip-components=1
+tar -xf zulip-server-latest.tar.gz
 ```
 
 Then, run the Zulip install script:
 ```
-/root/zulip/scripts/setup/install
+./zulip-server-*/scripts/setup/install
```

Comment:
I think we can probably combine the download/install steps at this point, right?